data/sql/updates/pending_db_world/rev_1646922359631359324.sql

```diff
@@ -0,0 +1,447 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1646922359631359324');
+
+DELETE FROM quest_template_locale where locale='zhTW' and id in(1657,1658,6961,6962,6963,6964,6983,6984,7021,7022,7023,7024,7025,7043,7045,7061,7062,7063,7790,8149,8150,8311,8312,8322,8353,8354,8355,8356,8357,8358,8359,8360,8373,8619,8635,8636,8642,8643,8644,8645,8646,8647,8648,8649,8650,8651,8652,8653,8654,8670,8671,8672,8673,8674,8675,8676,8677,8678,8679,8680,8681,8682,8683,8684,8685,8686,8688,8713,8714,8715,8716,8717,8718,8719,8720,8721,8722,8723,8724,8725,8726,8727,8744,8746,8762,8767,8768,8769,8788,8799,8803,8827,8828,8861,8862,8863,8864,8865,8866,8867,8868,8870,8871,8872,8873,8874,8875,8876,8877,8878,8879,8880,8881,8882,8883,8899,8900,8902,8903,8904,8971,8972,8973,8974,8975,8976,8979,8981,8983,8984,9027,9028,9319,9322,9323,9324,9326,9330,9331,9332,9339,9365,9367,9368,9386,9388,9389,11117,11118,11120,11122,11127,11131,11135,11219,11220,11242,11293,11294,11318,11320,11356,11357,11360,11361,11403,11405,11407,11408,11409,11412,11419,11431,11435,11439,11440,11441,11444,11445,11446,11447,11449,11450,11454,11528,11580,11581,11584,11732,11734,11735,11736,11737,11738,11739,11740,11741,11742,11743,11744,11745,11746,11747,11748,11749,11750,11751,11752,11753,11754,11755,11756,11757,11758,11759,11760,11761,11762,11763,11764,11765,11766,11767,11768,11769,11770,11776,11777,11778,11779,11780,11781,11782,11783,11784,11785,11786,11787,11799,11800,11801,11802,11803,11835,11836,11837,11838,11839,11840,11841,11842,11843,11844,11845,11846,11847,11848,11849,11850,11851,11852,11853,11854,11855,11856,11857,11858,11859,11860,11937,12361,12362,12363,12364,12365,12366,12367,12368,12369,12370,12371,12373,12374,12375,12376,12377,12378,12379,12380,12381,12382,12383,12384,12385,12386,12387,12388,12389,12390,12391,12392,12393,12394,12395,12396,12397,12398,12399,12400,12401,12402,12403,12404,12405,12406,12407,12408,12409,12410,12940,12941,12946,12947,12950,13065,13440,13441,13442,13443,13444,13445,13446,13447,13449,13450,13451,13452,13453,13454,13455,13456,13457,13458,13459,13460,13461,13462,13463,13464,13465,13466,13467,13468,13469,13470,13471,13472,13474,13493,13494,13495,13496,13497,13498,13499,13500,13501,13548,13952,13966,14022,14023,14024,14028,14030,14033,14035,14036,14037,14040,14041,14043,14044,14047,14048,14051,14053,14054,14055,14058,14059,14060,14061,14062,14064,14065,14166,14171,14172,14173,14174,14175,14176,14177,14483,14488,24536,24541,24576,24597,24609,24610,24611,24612,24613,24614,24615,24629,24635,24636,24645,24647,24650,24651,24652,24655,24656,24657,24659,24660,24662,24664,24665,24666,24745,24792,24793,24804,24805,24848,24849,24850,24851);
```

Comment:
Please add backticks around table and column names. Keywords in uppercase please